public/index.html

```diff
@@ -7,6 +7,6 @@
   <title>Document</title>
 </head>
```

Comment:
### Optional
- It would be better if we update our deployed link to reflect our current project